src/http/types/DaprPubSub.type.ts

```diff
@@ -1,3 +1,9 @@
 import { IRequest, IResponse } from "../lib/WebServer";
 
-export type TypeDaprPubSub = (req: IRequest, res: IResponse) => Promise<void>;
\ No newline at end of file
+export type TypeDaprPubSub = (req: IRequest, res: IResponse) => Promise<void>;
+export type TypeElementOfDaprPubSub = {
+    pubSubName:String,
```

Comment:
code formatting please :) space between the : and String (e.g. `pubSubName: string`)